target/linux/generic/backport-6.6/0080-v6.9-smp-Avoid-setup_max_cpus_namespace_collision_shadowing.patch

```diff
@@ -32,13 +32,16 @@ Cc: linux-kernel@vger.kernel.org
  #define cpuhp_tasks_frozen	0
 --- a/kernel/cpu.c
 +++ b/kernel/cpu.c
-@@ -1905,14 +1905,14 @@ static bool __init cpuhp_bringup_cpus_pa
+@@ -1905,17 +1905,17 @@ static bool __init cpuhp_bringup_cpus_pa
  static inline bool cpuhp_bringup_cpus_parallel(unsigned int ncpus) { return false; }
  #endif /* CONFIG_HOTPLUG_PARALLEL */
  
 -void __init bringup_nonboot_cpus(unsigned int setup_max_cpus)
 +void __init bringup_nonboot_cpus(unsigned int max_cpus)
  {
+ 	if (!setup_max_cpus)
```

Comment:
```suggestion
 	if (!max_cpus)
```
The upstream version of this patch does it too: https://github.com/torvalds/linux/commit/6ef8eb5125722c241fd60d7b0c872d5c2e5dd4ca

Reply:
> The upstream version of this patch does it too: [torvalds/linux@6ef8eb5](https://github.com/torvalds/linux/commit/6ef8eb5125722c241fd60d7b0c872d5c2e5dd4ca)

@graysky2 Please remove this upstreamed patch.
https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=v6.6.39&id=69787793e7f0673465c89714d1522fde978bbfa8